README.md

```diff
@@ -121,8 +121,7 @@ Thanks to Frame#5375 and AloXado320 for also helping with silhouette stuff
   - Use `render_multi_image` to draw large texture rectangles consisting of multiple images on the screen.
   - More info in `puppyprint.c`
 - Wiseguy's Farcall TLB mapping allows to store executable code inside uncompressed segments, that can be loaded and ran as needed, instead of it having to be loaded at all times. See `farcall.h` in the include folder for instructions and details.
-- Red Coin Stars now support up to 99 red coins! You can have red coin stars specific to areas, or you can use the hidden red coin star's 2nd behavior param to set a red coin
-count required to spawn the star.
+- Red Coin Stars now support up to 99 red coins! You can have red coin stars specific to areas, or you can use the hidden red coin star's 2nd behavior param to set a red coin count required to spawn the star.
```

Comment:
Area specific reds are possible as long as you don't set the 2nd bparam. Even though the gRedCoinsCollected variable would increment for both areas, either star will not spawn so long as any reds in its respective area still exist. Is it weird to have the reds count combined? I guess, but this use case is already niche as is.

Setting the 2nd bparam is specifically reserved for multi-area red coin missions since it uses a specific red coin number to spawn at. Otherwise you would use the vanilla method.

Reply:
but just in case, I clarified it in the readme

Reply:
Huh I guess it _technically_ would work, though yeah I wouldn't advertise it since the coin counter would be incorrect.